setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name='hm_pyhelper',
-    version='0.13.1',
+    version='0.13.2',
```

Comment:
Might need to bump it to `0.13.3` because of hm-pyhelper release I did yesterday.